react-ui/src/App.js

```diff
@@ -44,6 +47,10 @@ class App extends Component {
           {/* <Redirect from="/" to="dashboard/" /> */}
         </Route>
 
+        <Route path="/WorkerAvailableInputPage" component={WorkerInputpage}> 
```

Comment:
Hi Kyle,
Minor suggestion, but can the path be lowercase instead of uppercase to stay consistent with the rest of the routes, e.g. `inputAvailability` or something along that line?

Reply:
Adjusted the naming as requested